helix-core/src/main/java/org/apache/helix/util/RebalanceUtil.java

```diff
@@ -136,4 +141,17 @@
     result[1] = slaveStateValue;
     return result;
   }
+
+  public static void scheduleOnDemandPipeline(String clusterName, long delay) {
```

Comment:
We should validate both parameters, not just cluster name.

Reply:
Fixed.